web_src/js/components/DiffFileTreeItem.vue

```diff
@@ -1,40 +1,24 @@
 <template>
-  <div v-show="show" :title="item.name">
-    <!--title instead of tooltip above as the tooltip needs too much work with the current methods, i.e. not being loaded or staying open for "too long"-->
-    <div class="item" :class="[item.isFile ? 'filewrapper gt-p-1 gt-ac' : '', store.selectedItem === '#diff-' + item.file?.NameHash ? 'selected' : '']">
-      <!-- Files -->
-      <SvgIcon
-        v-if="item.isFile"
-        name="octicon-file"
-        class="svg-icon file"
-      />
-      <a
-        v-if="item.isFile"
-        :class="['file gt-ellipsis', {'viewed': item.file.IsViewed}]"
-        :href="item.isFile ? '#diff-' + item.file.NameHash : ''"
-      >{{ item.name }}</a>
-      <SvgIcon
-        v-if="item.isFile"
-        :name="getIconForDiffType(item.file.Type)"
-        :class="['svg-icon', getIconForDiffType(item.file.Type), 'status']"
-      />
+  <!--title instead of tooltip above as the tooltip needs too much work with the current methods, i.e. not being loaded or staying open for "too long"-->
+  <a
+    v-if="item.isFile" class="item-file"
+    :class="{'selected': store.selectedItem === '#diff-' + item.file.NameHash, 'viewed': item.file.IsViewed}"
+    :title="item.name" :href="'#diff-' + item.file.NameHash"
+  >
+    <!-- file -->
+    <SvgIcon name="octicon-file"/>
+    <span class="gt-ellipsis gt-f1">{{ item.name }}</span>
+    <SvgIcon :name="getIconForDiffType(item.file.Type).name" :class="getIconForDiffType(item.file.Type).classes"/>
+  </a>
+  <div v-else class="item-directory" :title="item.name" @click.stop="collapsed = !collapsed">
```

Comment:
Same here.

Reply:
Additionally, GitHub uses a `button` here which makes sense from an a11y perspective.
Perhaps we should do that too.
But let's leave that for another PR.